config/application.rb

```diff
@@ -69,5 +69,9 @@ class Application < Rails::Application
     config.force_jwt_at_registration = true
 
     config.feature_flag_mfa = ENV["FEATURE_FLAG_MFA"] == "enabled"
+
+    config.i18n.load_path += Dir[Rails.root.join("config/locales/**/*.yml").to_s]
+
+    config.i18n.default_locale = :en
```

Comment:
[This defaults to `:en`](https://guides.rubyonrails.org/configuring.html#configuring-i18n), so the config is unnecessary.

Reply:
any value in explicit over implicit config?